releases/v20.2.0-beta.1.md

```diff
@@ -0,0 +1,333 @@
+- `stall time` has been renamed to `IO time` in [`EXPLAIN ANALYZE`](../v20.2/explain-analyze.html) diagrams for queries executed via the [vectorized execution engine](../v20.2/vectorized-execution.html). [#53153][#53153]
+- Implemented geometry [builtin](../v20.2/functions-and-operators.html#built-in-functions) `ST_RemovePoint` [#53244][#53244]
+- An [`INSERT ... ON CONFLICT DO UPDATE`](../v20.2/insert.html#on-conflict-clause) statement without a list of column names after `ON CONFLICT` now results in a SQL syntax error with the error code `42601`. Previously, it errored with the message "there is no unqiue or exlcusion constraint matching the ON CONFLICT specification" and the error code `42P10`. [#53067][#53067]
+- Added support for the `WITH DATA` and `WITH NO DATA` arguments for the `REFRESH MATERIALIZED VIEW` command. [#53052][#53052]
+- Added support for the `GRANT ... ON SCHEMA command`. [#53344][#53344]
+- Added the `SHOW TYPES` command to list all user-defined types. [#53386][#53386]
+- Implemented the `ST_IsClosed`, `ST_IsSimple`, and `ST_IsRing` [builtins](../v20.2/functions-and-operators.html#built-in-functions). [#53240][#53240]
+
+### Command-line changes
+
+- The --storage-engine CLI flag no longer defaults to using the last-used storage engine when unspecified. Instead, pebble is used by default unless otherwise specified using the --storage-engine flag. [#52944][#52944]
+- Adds alias commands `ls` and `rm` for `userfile list` and `userfile delete`. [#53074][#53074]
+- The new debug command `decode-proto` reads descriptor from stdin in hex or base64 format (auto-detected) and a flag --schema=<fully qualified name to decode> with default value `cockroach.sql.sqlbase.Descriptor` and outputs to stdout the deserialized proto in JSON format. If the input is not a hex/base64 encoded proto then it is outputted verbatim. [#52972][#52972]
+- The `cockroach start` and `start-single-node` commands now enable `--redactable-logs` by default. It is also enabled by default in `cockroach demo` if `--log-dir` is passed. This causes log files to become redactable, so that `cockroach debug merge-log --redact` or `cockroach debug zip --redact` can remove sensitive information out of log files. (Reminder: `cockroach debug zip --redact` only affects *log files*; other items collected by the command can still contain sensitive information) [#53263][#53263]
+- The command-line flag `--socket` has been removed. It was deprecated since v20.1. Use `--socket-dir` in replacement. [#53405][#53405]
+- The command-line `--insecure` has been marked as deprecated. See https://go.crdb.dev/issue/53404 for details. The flag will be removed in a later version in a staged fashion: first, additional security mechanisms will be added to enable more flexible deployments which were previously done using `--insecure`; then the flaf will be removed from server commands, then finally in a later version also from client commands. [#53405][#53405]
```

Comment:
nits: 
- See https://go.crdb.dev/issue/53404 for details > See  issue [#53404](https://go.crdb.dev/issue/53404) for details
- flaf > flag